src/services/tourettes.js

```diff
@@ -62,12 +62,22 @@ function getProfanities(amount) {
   return profanityReply.join(' ');
 }
 
+function isInt(data) {
```

Comment:
Aaah, fair enough, then I only have one question, why not use Number.isInteger() instead of implementing it again? (Maybe it's just me missing something obvious :) )